api/libretime_api/legacy/migrations/0044_add_track_types_analyzer_options.py

```diff
@@ -0,0 +1,27 @@
+# pylint: disable=invalid-name
+
+from django.db import migrations
+
+from ._migrations import legacy_migration_factory
+
+UP = """
+alter table "cc_track_types" add column "analyze_cue_points" boolean default 't' not null;
+"""
```

Comment:
In the migration, the option is enabled by default, this is to prevent breaking with the previous behavior for existing track types, but it feels more 'normal' to have this disabled by default from now on ?

So we could set the column default to "disabled" but we enable all existing rows.

Reply:
I wondered the same thing, but I think having it disabled by default is better. Especially if it doesn't really work anyway